src/main/java/no/nav/familie/inntektsmelding/forespørsel/rest/LukkForespørselRequest.java

```diff
@@ -0,0 +1,14 @@
+package no.nav.familie.inntektsmelding.forespørsel.rest;
+
+import java.time.LocalDate;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+
+import no.nav.familie.inntektsmelding.typer.dto.OrganisasjonsnummerDto;
+import no.nav.familie.inntektsmelding.typer.dto.SaksnummerDto;
+
+public record LukkForespørselRequest(@NotNull @Valid OrganisasjonsnummerDto orgnummer,
```

Comment:
En tanke: Hvis vi gjør orgnr nullable så kunne vi brukt dette som endepunkt for å lukke alle oppgaver på saken når behandlingen avsluttes / vedtak fattes. Men dette er kanskje bedre å løse på annen måte?

Reply:
En bra tanke. Siden vi ikke vet helt enda så lar jeg den stå som Not Null, så kan vi heller endre når vi vet om det er løsningen vi går for